test/fixtures/musig.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
did you take these test vectors from somewhere? If yes, can you add a reference for the source to the issue description?

Reply:
I just made them up, to have something to ensure that I didn't break existing functions while developing.
I'll for proper test cases